src/StoreKeeper/WooCommerce/B2C/Imports/OrderImport.php

```diff
@@ -199,4 +210,22 @@ protected function getImportEntityName(): string
     {
         return __('orders', I18N::DOMAIN);
     }
+
+    /**
+     * @param $order
+     *
+     * @throws Exception
+     */
+    public static function fetchOrderStatusUrl(&$order, int $storekeeperId): void
```

Comment:
```suggestion
    public static function fetchOrderStatusUrl(<missing type> $order, int $storekeeperId): void
```

Using referenced for objects is deprecated and can you add type?

Reply:
Changed to pass by reference to `WC_Order` object.